aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/advanced-entity-framework-scenarios-for-an-mvc-web-application.md

```diff
@@ -329,5 +288,39 @@ Solution
 
 Check the connection string. If you have manually deleted the database, change the name of the database in the construction string.
 
-> [!div class="step-by-step"]
-> [Previous](implementing-inheritance-with-the-entity-framework-in-an-asp-net-mvc-application.md)
+## Get the code
+
+[Download Completed Project](http://code.msdn.microsoft.com/ASPNET-MVC-Application-b01a9fe8)
+
+## Additional resources
+
+ For more information about how to work with data using the Entity Framework, see the [EF documentation page on MSDN](https://msdn.microsoft.com/data/ee712907) and [ASP.NET Data Access - Recommended Resources](../../../../whitepapers/aspnet-data-access-content-map.md).
+
+For more information about how to deploy your web application after you've built it, see [ASP.NET Web Deployment - Recommended Resources](../../../../whitepapers/aspnet-web-deployment-content-map.md) in the MSDN Library.
+
+For information about other topics related to MVC, such as authentication and authorization, see the [ASP.NET MVC - Recommended Resources](../recommended-resources-for-mvc.md).
+
+## Next steps
+
+This completes this series of tutorials on using the Entity Framework in an ASP.NET MVC application. You can continue learning about Entity Framework in the next node.
```

Comment:
This might work better after he bullet lists rather than before them.

As this is the last in the series, there isn't a next node. Could replace this sentence with one like the suggested text in the next comment.